src/domain/place/placeInfo/PlaceInfo.tsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+import { usePlaceQuery } from '../../../generated/graphql';
+import { Language } from '../../../types';
+import getLocalizedString from '../../../utils/getLocalisedString';
+import VenueInfo from '../../venue/venueInfo/VenueInfo';
+import { generateHslLink, generateServiceMapLink } from '../utils';
+import styles from './placeInfo.module.scss';
+
+interface Props {
+  id: string;
+  language: Language;
+  onEditButtonClick?: (show: boolean) => void;
+  showEditButton?: boolean;
+  showVenueInfo?: boolean;
+}
+
+const PlaceInfo: React.FC<Props> = ({
+  id,
+  language,
+  onEditButtonClick,
+  showEditButton,
+  showVenueInfo,
+}) => {
+  const { t } = useTranslation();
+  const { data } = usePlaceQuery({ variables: { id } });
+
+  if (!data) return null;
+
+  const name = getLocalizedString(data.place?.name || {}, language);
+  const streetAddress = getLocalizedString(
+    data.place?.streetAddress || {},
+    language
+  );
+  const telephone = getLocalizedString(data.place?.telephone || {}, language);
+  const addressLocality = getLocalizedString(
+    data.place?.addressLocality || {},
+    language
+  );
+  const serviceMapLink = generateServiceMapLink(id, language);
+  const hslLink = generateHslLink(streetAddress, addressLocality, language);
```

Comment:
Maybe we could have getPlaceFields (similar as getEventFields) helper function to get all these values.